src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Processors/GeneralizeProcessor.cs

```diff
@@ -19,7 +19,14 @@ public ProcessResult Process(ElementNode node, ProcessContext context = null, Di
             EnsureArg.IsNotNull(settings);
 
             var result = new ProcessResult();
-            if (!ModelInfo.IsPrimitive(node.InstanceType) || node.Value == null)
+            if (!ModelInfo.IsPrimitive(node.InstanceType))
+            {
+                throw new AnonymizerRuleNotApplicableException(
+                    $"Generalization is not applicable on the node with type {node.InstanceType}. " +
```

Comment:
you can remove "+"

Reply:
Just want to split code into two lines. I removed it.